src/ai/threads/messages/message.ts

```diff
@@ -114,7 +118,151 @@ export default class Messages {
     return { ...txResult, messages };
   }
 
-  private async getMessageCreateTxBody(
+  async update(
+    threadId: string,
+    messageId: string,
+    { objectId, tokenId, provider, api, metadata }: MessageUpdateParams
+  ): Promise<MessageTransactionResult> {
+    const appId = Ainft721Object.getAppId(objectId);
+    const address = this.ain.signer.getAddress();
+
+    await validateObject(appId, this.ain);
+    await validateToken(appId, tokenId, this.ain);
+
+    const aiName = validateAndGetAiName(provider, api);
+    await validateAiConfig(appId, aiName, this.ain);
+    await validateTokenAi(appId, tokenId, aiName, null, this.ain);
+
+    await validateThread(appId, tokenId, aiName, address, threadId, this.ain);
+    await validateMessage(
+      appId,
+      tokenId,
+      aiName,
+      address,
+      threadId,
+      messageId,
+      this.ain
+    );
+
+    const message = <ThreadMessage>{
+      id: 'msg_000000000000000000000001',
```

Comment:
(question) messageId or just a mock data for further updates? if in the latter case, a todo?

Reply:
yes, this mock data is replaced with ainize method after deploy. I'll add todo comment thanks!

Reply:
done!